### PR TITLE
Lara Hilborn

### PR DESCRIPTION
**Pull request by LaraHilborn**

### CODE REVIEW

File: css/index.css [hunk 1]
```diff
@@ -87,34 +102,174 @@ p {
...
+    background: teal;
+    margin: 20px 2.5%;
+    color: white;
+    display: flex;
+    align-items: center;
+    justify-content: center;
+   
+}
+
+.middle-content .boxes div + div {
```
Comment (nathan-loveless) [inline css/index.css:138]: On each individual box, the parent can style all the individual boxes, the only thing that needs to be done on the boxes is to override the color.

File: home.html [hunk 2]
```diff
@@ -12,8 +12,28 @@
 </head>
 
 <body>
+
+    <!----header-->
```
Comment (nathan-loveless) [inline home.html:16]: The index.html page is looking good.  One thing you can do it make the header and nav the same size with the rest of the content page like you did with the about.html.

File: about.html [hunk 3]
```diff
@@ -10,64 +10,106 @@
...
     
-    How We Work
+            <h1>Strategy</h1>
     
-    Iterative approaches to corporate strategy foster collaborative thinking to further the overall value proposition.
+            Iterative approaches to corporate strategy foster collaborative thinking to further the overall value proposition.
 
-    Learn More
+            <button href="#">Learn More</button>
+        </div>
```
Comment (nathan-loveless) [inline about.html:54]: The only thing it looks like you are missing are the dashed bottom border lines on each of these 4 sections.

File: css/index.css (hunk 1 above)
Comment (LaraHilborn) [reply css/index.css:138]: Ah, ok. I knew there had to be an easier way to get that to work. 
Thank you

**Pull request closed without merging**